packages/blade/.babelrc.js

```diff
@@ -30,6 +30,11 @@ const configs = {
             alias: {
               '@storybook/react': '@storybook/react-native',
               '^styled-components$': 'styled-components/native',
+              // react-native path resolution
+              '~src': './src',
+              '~components': './src/components',
+              '~utils': './src/utils',
+              '~tokens': './src/tokens',
```

Comment:
Can we directly name the folder instead of adding `~`?
```suggestion
              'src': './src',
              'components': './src/components',
              'utils': './src/utils',
              'tokens': './src/tokens',
```

Reply:
We had done this on mobile app repo. Can you check that implementation once and see if we can replicate it?

Ref: 
- https://github.com/razorpay/frontend-mobile-app/blob/master/.eslintrc.js#L40
- https://github.com/razorpay/frontend-mobile-app/blob/master/.babelrc.js#L25

Reply:
I've already done it for babelrc. 

And for eslint we are using typescript module resolver so it works out of the box by referencing the tsconfig paths.